src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -136,7 +136,7 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
                 .FirstOrDefault();
 
             // installVersion is null if the package is not installed
-            var installedVersion = installedDependency?.VersionRange?.MinVersion;
+            var installedVersion = installedDependency?.VersionRange;
```

Comment:
Both, there was no need to have `installedVersion` since we already have a global `InstalledVersions` that is not the minimum of a version range, but I found a bug while doing the tests (sorry for not pointing it out in the description) below I was comparing `!latestStableVersion.version.Equals(installedVersion)` where if `latestStableVersion` was `3.0.0` and installedVersion was `3.*`, doind the MinVersion will convert it to `3.0.0` and mess with the filtering later